web_monitoring/diffing_server.py

```diff
@@ -48,8 +61,23 @@ def get(self, differ):
         a = query_params.pop('a')
         b = query_params.pop('b')
 
-        # Fetch server response for URLs a and b.
-        res_a, res_b = yield [client.fetch(a), client.fetch(b)]
+        # Special case for local files, for dev/testing.
+        if a.startswith('file://') and b.startswith('file://'):
+            if os.get('WEB_MONITORING_APP_ENV') in ('development', 'testing'):
+                headers = {'Content-Type': 'application/html; charset=UTF-8'}
+                with open(a[7:], 'rb') as f:
+                    body = f.read()
+                    res_a = MockResponse(a, body, headers)
+                with open(b[7:], 'rb') as f:
+                    body = f.read()
+                    res_b = MockResponse(b, body, headers)
+            else:
+                self.send_error(
+                    500, reason=("Local files can only be used in development "
+                                 "or testing environment."))
```

Comment:
I think you need a `return` here, right? Otherwise the server continues on to explode on line 83 after sending the response.